api/resource/factory.go

```diff
@@ -142,6 +143,19 @@ func (rf *Factory) SliceFromBytes(in []byte) ([]*Resource, error) {
 	return result, nil
 }
 
+// yaml.v3 panics on multibyte characters, so we must remove them from the inputs
```

Comment:
does this drop data as well as comments?

Reply:
you're right, this will drop data that uses a multibyte character not in a comment. I had assumed the case of a multibyte character as data would panic as well but it seems to work fine. I will add a regression test for that and close this PR.